### PR TITLE
Configure incrementals

### PR DESCRIPTION
**Pull request by varyvol**
Configure incrementals so that incremental releases are published on each build.

### CODE REVIEW

**Comment by jglick** [comment]
CI failure, retrying…

**Comment by jglick** [comment]
Seems the new EC2 agents are pretty broken.

**Comment by jglick** [comment]
[INFRA-2548](https://issues.jenkins-ci.org/browse/INFRA-2548)

**Comment by jglick** [comment]
retrying…

**Comment by varyvol** [comment]
@jglick seems to be passing now...

File: pom.xml [hunk 1]
```diff
@@ -8,17 +8,19 @@
...
     <name>Jenkins Mercurial plugin</name>
     <description>Allows Jenkins to check out projects from the Mercurial SCM.</description>
     <url>https://wiki.jenkins.io/display/JENKINS/Mercurial+Plugin</url>
     
     <properties>
-      <jenkins.version>2.150.3</jenkins.version>
-      <java.level>8</java.level>
-      <no-test-jar>false</no-test-jar>
-      <reuseForks>false</reuseForks> <!-- long story -->
+        <revision>2.9</revision>
```
Comment (jglick) [inline pom.xml:18]: Bad merge conflict resolution:

```suggestion
        <revision>2.10</revision>
```

Tip:

```ini
[merge]
	conflictstyle = diff3
```

**Comment by jglick** [comment]
Need a new CI build…

**Comment by varyvol** [comment]
@jglick passing now.

**Pull request merged**